src/test.js

```diff
@@ -674,66 +674,122 @@ function checkPollution() {
 	}
 }
 
+function addTestWithData( data ) {
```

Comment:
I like this abstraction a lot, did you mean to make more use of it? I went ahead and did so as part of https://github.com/qunitjs/qunit/pull/1620.

Reply:
This actually came from @smcclure15's [recommendation](https://github.com/qunitjs/qunit/pull/1569#discussion_r608754046). Glad it's useful!